HeterogeneousCore/CUDAUtilities/interface/eigenSoA.h

```diff
@@ -0,0 +1,55 @@
+#ifndef HeterogeneousCore_CUDAUtilities_interface_eigenSoA_h
+#define HeterogeneousCore_CUDAUtilities_interface_eigenSoA_h
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+
+#include <Eigen/Core>
```

Comment:
The header consistency tool complains missing header. I presume this is because the BuildFile does not declare `<use name="eigen"/>`. Would adding that dependence have any ill effects?

Reply:
I think it makes sense to add it, even just for having the transitive dependency in packages using this one.